shared/pkg/utils/trivy/sourcetypes.go

```diff
@@ -27,6 +31,8 @@ func KubeclaritySourceToTrivySource(sourceType utils.SourceType) (artifact.Targe
 	switch sourceType {
 	case utils.IMAGE:
 		return artifact.TargetContainerImage, nil
+	case utils.DOCKERARCHIVE, utils.OCIARCHIVE:
```

Comment:
In case of OCIARCHIVE you untar the tar file. Should the type be TargetFilesystem instead (same as DIR)?

Reply:
When you unpack a OCI archive you don't get a root file system, you get the contents of a container image (layers etc) represented as a folder structure this is what trivy ingests.